node/test/go/dev-mode/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
I think we do not need this file.  Let's just add `test/**` here: https://github.com/serverless/console/blob/ce3e992f6322458519ec9b53e4c1ba25b3f29778/node/.eslintrc.js#L10

Reply:
Still, not addressed

Reply:
@medikoo I had tagged you in a comment about this, [If I exclude this file the lint process fails.](https://github.com/serverless/console/pull/188/commits/dd2cb9c1217a63454175c1bee3be6d48e88f65d6)

Reply:
Please remove it, and let me see what the fail is.

If anything needs to be adjusted in lint configuration, we should do it in `node/.eslintrc.js` and not introduce a new one here

Reply:
[You can see the errors on the build when I removed it](https://github.com/serverless/console/actions/runs/3108502202/jobs/5037805345). I wasn’t sure what it needed.

Reply:
Back then errors were caused by `package.json` being in this folder (ESLint rules expected dev deps to be defined in its context), now it's no longer the case.

Reply:
Ahh okay got it 👍 Seems to be working now.